src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,141 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { buildHybridComponent } from '../../util/state-management';
+import {
+	ExpanderPanelDumb as ExpanderPanel,
+	IExpanderPanelHeaderProps,
+} from '../ExpanderPanel/ExpanderPanel';
+import {
+	findTypes,
+	omitProps,
+	StandardProps,
+	Overwrite,
+} from '../../util/component-types';
+import * as reducers from '../Accordion/Accordion.reducers';
+
+const cx = lucidClassNames.bind('&-Accordion');
+
+const { any, func, object, number, string } = PropTypes;
+
+interface IAccordionPropsRaw extends StandardProps {
+	/**
+	 * Indicates which item is expanded
+	 * */
+	selectedIndex: number;
+	/**
+	 * Called when the user clicks on the component's header of an item.
+	 * */
+
+	onSelect: (
+		selectedIndex: number | null,
+		{ event, props }: { event: React.MouseEvent; props: IAccordionProps }
+	) => void;
+	/**
+	 * Prop alternative to Header child component passed through to the
+	 * underlying ExpanderPanel
+	 */
+	Header: IExpanderPanelHeaderProps;
+}
+
+type IAccordionProps = Overwrite<
+	React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+	IAccordionPropsRaw
+>;
+
+export interface IAccordionState {
+	selectedIndex: number | null;
+}
+
+const defaultProps = {
+	onSelect: _.noop,
+};
+
+class Accordion extends React.Component<IAccordionProps, IAccordionState> {
+	static displayName = 'Accordion';
+	static propTypes = {
+		className: string`
+			Appended to the component-specific class names set on the root element.
+		`,
+		selectedIndex: number`
+			Indicates which item is expanded
+		`,
+		onSelect: func`
+			Called when the user clicks on the component's header of an item.
+		`,
+		style: object`
+			Passed through to the root element.
+		`,
+		Header: any`
+			Prop alternative to Header child component passed through to the
+			underlying ExpanderPanel
+		`,
+	};
+	static definition = {
```

Comment:
Shouldn't need this `definition` object anymore with `buildModernHybridComponent`. However you will want to stick the `peek` property on as a class static.

Reply:
Ah good to know. I converted it to a functional component when I added `buildModernHybridComponent`,  so it has the right static property `peek` now.